app/directives/meetings/admeetings.directive.html

```diff
@@ -3,7 +3,10 @@
     <div class="btn-group btn-group-justified">
 
         <div class="btn-group" uib-dropdown is-open="agencyIsOpen">
```

Comment:
Couldn't agencyIsOpen be instead ctrl.agencyIsOpen? 
And in controller self.agencyIsOpen instead of $scope.agencyIsOpen.
Same for all is-opens.